client/app/components/permissions-editor/index.js

```diff
@@ -53,7 +53,14 @@ const PermissionsEditorComponent = {
       $http.post(this.aclUrl, body).success(() => {
         user.alreadyGrantee = true;
         loadGrantees();
-      });
+      })
+        .catch((error) => {
+          if (error.status === 403) {
+            toastr.error('You can not add a user to this dashboard. Ask the dashboard owner to grant them permissions.');
```

Comment:
I think it needs to be `cannot` and not `can not`, right?

Reply:
Oxford Dictionary says both are acceptable: https://en.oxforddictionaries.com/usage/cannot-or-can-not However, since I had to do a commit to address your second comment, I changed it.